doc/FEATURES_RELEASE_2.md

```diff
@@ -41,6 +41,9 @@ DDI. Press `Submit`. The system should display the DDI information
 for the DDI found, including whether its disposition is `Good` or `Bad`,
```

Comment:
By reading line 44, I think this should say `Harmful` rather than bad. `Good` probably needs to be made consistent too.